dotnet/DatabaseMigration/DatabaseMigration.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="MySql.Data" Version="8.0.23" />
+    <PackageReference Include="Pulumi.Automation" Version="2.22.0-alpha.1614645792" />
```

Comment:
```suggestion
    <PackageReference Include="Pulumi.Automation" Version="2.22" />
```

Reply:
This is a result of me writing these examples on my own machine. I don't have access to any release version of `Pulumi.Automation` so I can't run them locally as `2.22`

Reply:
Yup. Makes sense. 2.22 is going out as we speak, so I think it'll be safe to update these references when we merge.

Reply:
The one that went out today was `2.22.0-preview` btw - should be good now though

Reply:
Ah, yes, my mistake. Of course :) We do still have it publishing under the preview flag.